integration_tests/test_asset_group_sightings.py

```diff
@@ -326,6 +326,7 @@ def test_asset_group_sightings(session, login, codex_url, test_root):
     sighting_guid = response.json()['guid']
     assert response.json() == {
         'guid': sighting_guid,
+        'id': sighting_guid,
```

Comment:
I'm in two minds about this, it would be really simple to remove it and as the frontend doesn't use it, I think we should. Any thoughts?